packages/react/src/components/Panel/index.tsx

```diff
@@ -69,7 +83,10 @@ Panel.propTypes = {
   children: PropTypes.node.isRequired,
   // @ts-expect-error
   heading: PropTypes.oneOfType([PropTypes.object, PropTypes.node]),
-  className: PropTypes.string
+  className: PropTypes.string,
+  padding: PropTypes.bool
 };
 
+export { PanelHeader, PanelContent };
```

Comment:
Just curious, why import/re-export here? We've typically used just an index component for this.

Reply:
I think I originally had everything in this single file, then broke it out into separate files. Are you thinking/do you prefer I do rename this to `Panel.tsx`, then import/export everything from a `index.tsx`?

Reply:
The 2 ways we've done it:

- Export everything via the `src/index.ts` file
- Export via an `component/${name}/index.ts` file.

I don't have a preference, just didn't want to introduce a new variant 🙃